card_db_src/de/sets_de.json

```diff
@@ -240,6 +240,11 @@
         "set_text": "",
         "text_icon": "R"
     },
+    "risingsun": {
+        "set_name": "Rising Sun",
+        "set_text": "Wir segeln jetzt zu den Inseln im Osten - oder Westen, das hängt davon ab, wo du dich im Verhältnis zu ihnen befindest. Dein Titel ist hier \"Kaiser\". Man sagte dir, du bist zwar nur ein Grüßaugust, aber du kannst trotzdem zum Frühstück alles bestellen, was du willst. Vielleicht haben sie recht, du hast dein gewünschtes Frühstück bekommen. Dein Zeremonienschwer ist nur aus Pappe, genau wie die Rüstung. Die Samurai lassen dich nie zu ihren Teepartys und die Ninjas binden immer deine Schnürsenkel zusammen. Und das epische Gedicht, das sie dir widmeten, hat nur 17 Silben. Reis wird jetzt als Währung benutzt, und niemand versucht auch nur, dein Gesicht auf die Körner zu bekommen. Aber wenn du morgens aufwachst und über dein Land schaust, findest du dein Leben gar nicht so schlecht. Also, was gibt's heute zum Frühstück?",
```

Comment:
Zeremonienschwer -> Zeremonienschwert

Reply:
Ah, in der Tat, da fehlt was. Soll ich das ändern und nochmals commiten, oder warten, ob noch ein weiterer Typo auftaucht?

Reply:
Ich denke einfach kurz verbessern und dann werde ich ein Deployment machen, falls andere später Typos finden können die dann korrigiert werden.